src/main/webapp/components/results/result-index-cards.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import { IndexCardItem, Actions } from '../index-cards'
+import AddIcon from '@material-ui/icons/Add'
+import GetAppIcon from '@material-ui/icons/GetApp'
+import IconButton from '@material-ui/core/IconButton'
+import Popover from '@material-ui/core/Popover'
+import useAnchorEl from '../../react-hooks/use-anchor-el'
+import { ResultListInteraction } from '../lists/result-list-interaction'
+
+export default props => {
+  const { results, setLists, lists } = props
+  const [selectedResult, setSelectedResult] = React.useState(null)
+  const [anchorEl, handleOpen, handleClose, isOpen] = useAnchorEl()
+
+  return (
+    <React.Fragment>
+      {' '}
```

Comment:
should this be deleted?

Reply:
I'm not sure why but whenever I format this comes up whenever something is wrapped in a React.Fragment. I'll remove it.